src/gc.c

```diff
@@ -79,7 +79,15 @@ static void threadCallback(void* data) {
   GCTask* task= data;
   GCContext* gc = task->gc;
   RedisModuleBlockedClient* bc = task->bClient;
-  RedisModuleCtx* ctx = RSDummyContext;
+  RedisModuleCtx* ctx;
+  switch (RSGlobalConfig.gcPolicy) {
+    case GCPolicy_Fork: 
+      ctx = ((ForkGC *)(gc->gcCtx))->ctx;
```

Comment:
I believe there is no harm in creating ThreadSafeCtx anyway each time.

Reply:
Replaced with `RedisModule_GetThreadSafeContext` and freed.
Let's push this.